docker/test.d/100_default.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+export PUBLIC_YVES_DOMAIN=http://yves
```

Comment:
What if the environment changes and yves isn't available via `yves`? To support this case, we need something like `export PUBLIC_YVES_DOMAIN="http://$PUBLIC_YVES_DOMAIN"`.

Please also fix the next statement